cmd/minishift/cmd/config/config.go

```diff
@@ -77,6 +77,9 @@ var (
 	OpenshiftEnv      = createFlag("openshift-env", nil, nil, nil, false)
 	Metrics           = createFlag("metrics", SetBool, nil, nil, true)
 	Logging           = createFlag("logging", SetBool, nil, nil, true)
+	ServiceCatalog    = createFlag("service-catalog", SetBool, nil, nil, true)
+	// ability to specify additional cluster up flags
+	OpenshiftExtraFlags = createFlag("openshift-extra-flags", SetString, nil, nil, true)
```

Comment:
I would request it to name as `OcClusterExtraFlags` because at first look `OpenshiftExtraFlags` is something we pass to openshift container.